source/content/guides/composer-convert.md

```diff
@@ -12,151 +12,142 @@ reviewed: "2021-03-10"
 
-1.  Copy any existing configuration from the default branch. If no files are copied through this step, that's ok:
+## Add the Pantheon Integrated Composer Upstream in a New Local Branch
 
-    ```bash{promptUser:user}
-    git checkout master sites/default/config
-    git mv sites/default/config/* config
-    git rm -f sites/default/config/.htaccess
-    git commit -m "Pull in configuration from default branch"
-    ```
+This process involves significant changes to the codebase. We recommend you to do this work on a new branch, as it might take you some time to complete and rolling back changes can be complicated:
 
-1.  Check for `pantheon.yml` settings you need to preserve by comparing your old codebase's `pantheon.yml` to the new `pantheon.upstream.yml`:
+1. In your local terminal, change directories to the site project. For example, if you keep your projects in a folder called `projects` in the home directory:
 
-    ```bash{promptUser:user}
-    git diff master:pantheon.yml pantheon.upstream.yml
-    ```
+  ```bash{promptUser:user}
+  cd ~/projects/$SITE/
+  ```
+
+1. Add the Pantheon Drupal Upstream as a new remote called `ic`, fetch the `ic` branch, and checkout to a new local branch based on it called `composerify`:
+
+  ```bash{promptUser:user}
+  git remote add ic git@github.com:pantheon-upstreams/drupal-project.git && git fetch ic && git checkout -b composerify ic/master
+  ```
+
+  If you prefer, you can replace `composerify` with another branch name. If you do, remember to adjust the other examples in this doc to match.
 
-- If there are settings from `pantheon.yml` (shown with a `-` in the diff output), consider copying over your old `pantheon.yml` to preserve these settings:
+1. Copy any existing configuration from the default branch. If no files are copied through this step, that's ok:
 
   ```bash{promptUser:user}
-  git checkout master pantheon.yml
-  git add pantheon.yml
-  git commit -m 'Copy my pantheon.yml'
+  git checkout master sites/default/config
+  git mv sites/default/config/* config
+  git rm -f sites/default/config/.htaccess
+  git commit -m "Pull in configuration from default branch"
   ```
 
-If you prefer to keep the value for `database` from `pantheon.upstream.yml`, remove it from `pantheon.yml`.
+1. Check for `pantheon.yml` settings you need to preserve by comparing your old codebase's `pantheon.yml` to the new `pantheon.upstream.yml`:
+
+  ```bash{promptUser:user}
+  git diff master:pantheon.yml pantheon.upstream.yml
+  ```
+
+   - If there are settings from `pantheon.yml` (shown with a `-` in the diff output), consider copying over your old `pantheon.yml` to preserve these settings:
+
+     ```bash{promptUser:user}
+     git checkout master pantheon.yml
+     git add pantheon.yml
+     git commit -m 'Copy my pantheon.yml'
+     ```
+
+   If you prefer to keep the value for `database` from `pantheon.upstream.yml`, remove it from `pantheon.yml`.
```

Comment:
Most users will not have a `database` entry in their pantheon.yml file. The advice we give here varies based on when this guide is published. If the restriction on Mariadb upgrades has been lifted, then we should not have users modifying the database entry at all. If someone wants to convert before the restriction has been lifted, then they *must* override the database entry in their pantheon.yml and pin it back to 10.0, but if they do this then they *must* come back in the future and remove their pin.

Given the timeframe, it's probably best to wait until the restriction is lifted before publishing.